srcpkgs/tpm2-tools/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'tpm2-tools'
+pkgname=tpm2-tools
+version=5.2
+revision=1
+build_style=gnu-configure
+hostmakedepends="autoconf autoconf-archive automake libtool pandoc pkg-config"
```

Comment:
pandoc is not available on all architectures, could you use it only on some architectures like for example in the bird_exporter package? The same applies to tpm2-totp.

Reply:
Should be good now, but please take another look